doc/guides/contributing/pull-requests.md

```diff
@@ -251,7 +251,9 @@ And on Windows:
 > vcbuild test
 ```
 
-(See the [running tests][] section of Building guide for more details.)
+(This code might take more than an hour to execute on a PC, want to run a part
+of tests quickly or for more details see the [running tests][] section of Buil
+ding guide.)
```

Comment:
```suggestion
of tests quickly or for more details see the [running tests][] section of
Building guide.)
```

Reply:
That is a very creative way to appease the linter's max-length, but please split sentences between words rather than splitting words between letters. 😉

Reply:
Thank you for reminding me that I will pay attention in the future